src/PrestaShopBundle/Resources/views/Admin/Common/Grid/grid.html.twig

```diff
@@ -0,0 +1,217 @@
+                     name="{{ gridView.identifier }}_bulk[]"
+                     type="checkbox"
+                     title=""
+                     value=""
+              >
+            </th>
+          {% endif %}
+
+          {# Render row columns here #}
+          {% for column in gridView.columns %}
+            {% set content = row.data[column.identifier] %}
+
+            {% if column.is_raw %}
+              <td>{{ content|raw }}</td>
+            {% else %}
+              <td>{{ content }}</td>
+            {% endif %}
+          {% endfor %}
+
+          {# If row has any actions then render those here #}
+          {% set rowActions = row.actions %}
+          {% if rowActions is not empty %}
+            <td class="text-right">
+              <div class="btn-group-action">
+                <div class="btn-group">
+                  <a href="{{ rowActions|first.url }}"
+                     title="{{ rowActions|first.name }}"
+                     class="btn tooltip-link"
+                     id="{{ '%s_grid_row_action_%s'|format(gridView.identifier, rowActions|first.identifier) }}"
+                  >
+                    <i class="material-icons">{{ rowActions|first.icon }}</i>
+                  </a>
+
+                  {# If there's more than one action, then render remaining actions here #}
+                  {% if rowActions|length > 1 %}
+                    <a class="btn btn-link dropdown-toggle dropdown-toggle-split"
+                       data-toggle="dropdown"
+                       aria-haspopup="true"
+                       aria-expanded="false"
+                    >
+                    </a>
+
+                    <div class="dropdown-menu dropdown-menu-right">
+                      {% for rowAction in rowActions|slice(1) %}
+                        <a class="dropdown-item"
+                           href="{{ rowAction.url }}"
+                           id="{{ '%s_grid_row_action_%s'|format(gridView.identifier, rowAction.identifier) }}"
+                        >
+                          <i class="material-icons">{{ rowAction.icon }}</i>
+                          {{ rowAction.name }}
+                        </a>
+                      {% endfor %}
+                    </div>
+                  {% endif %}
+                </div>
+              </div>
+            </td>
+          {# If filters are available, it means we have "Search" button, then add empty column to fill the row #}
+          {% elseif filtersForm|length > 1 %}
+            <td>&nbsp;</td>
+          {% endif %}
+        </tr>
+      {% endfor %}
+    {% else %}
+      <tr>
+        <td colspan="{{ gridView.columns|length }}">
+          <div class="text-center">
+            <p><i class="material-icons">warning</i></p>
+            <p>{{ 'No records found'|trans }}</p>
+          </div>
+        </td>
+      </tr>
+    {% endif %}
+    </tbody>
+  </table>
+
+  {% if gridView.data.rows_total > 10 %}
```

Comment:
I'm pretty sure this value should come from grid configuration (the "10"), wdyt?

Reply:
it makes sense, but where should it be stored? should it be configured per grid or it's a global value for all grids?

Reply:
imo for each grid, so we may complete grid definition here, isn't it?